tests/instancetype_test.go

```diff
@@ -779,6 +779,62 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			}, 5*time.Minute, time.Second).Should(Succeed())
 		})
 
+		It("[test_id:10161] should apply preferences to termination grace period", func() {
+			var numOfSecondsForTest int64 = 300
+			vmi := libvmi.NewCirros()
+
+			clusterPreference := newVirtualMachineClusterPreference()
+			clusterPreference.Spec.PreferredTerminationGracePeriodSeconds = pointer.Int64(numOfSecondsForTest)
+
+			clusterPreference, err := virtClient.VirtualMachineClusterPreference().
+				Create(context.Background(), clusterPreference, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			vm := tests.NewRandomVirtualMachine(vmi, false)
+			vm.Spec.Preference = &v1.PreferenceMatcher{
+				Name: clusterPreference.Name,
+			}
+
+			vm, err = virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm)
+			Expect(err).ToNot(HaveOccurred())
+
+			vm = tests.StartVMAndExpectRunning(virtClient, vm)
+			vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Get(context.Background(), vm.Name, &metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			fmt.Printf("value is %d\n", vmi.Spec.TerminationGracePeriodSeconds)
+			fmt.Printf("value is %d\n", numOfSecondsForTest)
```

Comment:
These `Printf`s are not needed, because if the test fails, the below `Expect()` will print the values.

Reply:
This was a left over thanks.